@here/harp-mapview/lib/DataSource.ts

```diff
@@ -209,13 +206,26 @@ export abstract class DataSource extends THREE.EventDispatcher {
     /**
      * Invoked by [[MapView]] to notify when the [[Theme]] has been changed.
      *
-     * If `DataSource` depends on a theme, it must update its tiles' geometry.
+     * If `DataSource` depends on a `styleSet` or `languages`, it must update its tiles' geometry.
+     *
+     * @deprecated, Use [[setTheme]].
      *
      * @param styleSet The new theme that [[MapView]] uses.
      * @param languages An optional list of languages for the `DataSource`.
      */
     // tslint:disable-next-line:no-unused-variable
-    setStyleSet(styleSet?: StyleSet, languages?: string[]): void {
+    setStyleSet(styleSet?: StyleSet, definitions?: Definitions, languages?: string[]): void {
```

Comment:
I believe this is still used in a lot of our public examples. Don't say that the examples update should be in the same change, but please create a ticket to tackle these in the future.

Reply:
Following up in #830